Tests/Common/Exceptions/StackExceptionInterpreterTests.cs

```diff
@@ -28,8 +28,8 @@ public class StackExceptionInterpretersTests
         public void CreatesFromAssemblies()
         {
             var assembly = typeof(FakeExceptionInterpreter).Assembly;
-            var projector = StackExceptionInterpreter.CreateFromAssemblies(new[] {assembly});
-            Assert.AreEqual(1, projector.Interpreters.Count(p => p.GetType() == typeof(FakeExceptionInterpreter)));
+            var interpreter = StackExceptionInterpreter.CreateFromAssemblies(new[] {assembly});
+            Assert.AreEqual(1, interpreter.Interpreters.Count(p => p.GetType() == typeof(FakeExceptionInterpreter)));
```

Comment:
thanks! love seeing the [boy scout rule](http://programmer.97things.oreilly.com/wiki/index.php/The_Boy_Scout_Rule) alive and well :D